lib/FieldType/Mapper/SelectionFormMapper.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace EzSystems\RepositoryForms\FieldType\Mapper;
+
+use EzSystems\RepositoryForms\Data\FieldDefinitionData;
+use EzSystems\RepositoryForms\FieldType\DataTransformer\CountryValueTransformer;
+use EzSystems\RepositoryForms\FieldType\FieldTypeFormMapperInterface;
+use Symfony\Component\Form\Extension\Core\ChoiceList\ChoiceList;
+use Symfony\Component\Form\FormInterface;
+
+class SelectionFormMapper implements FieldTypeFormMapperInterface
+{
+    public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, FieldDefinitionData $data)
+    {
+        $fieldDefinitionForm
+            ->add('isMultiple', 'checkbox', [
+                'required' => false,
+                'property_path' => 'fieldSettings[isMultiple]',
+                'label' => 'field_definition.ezselection.is_multiple',
+            ])
+            ->add('options', 'collection', [
+                'type' => 'text',
+                'options' => [
+                    'required' => false,
+                ],
+                'allow_add' => true,
+                'allow_delete' => true,
+                'delete_empty' => true,
+                'prototype' => true,
+                'prototype_name' => mt_rand(),
```

Comment:
This is the bit that should be fixed. Ideally it should be an incrementing number based on the number of selection entries, but we don't know that at this point, hence the large random number. Anything unique works, a collision would cause trouble.